base/src/com/google/idea/blaze/base/projectview/ProjectView.java

```diff
@@ -129,6 +130,10 @@ public <T, SectionType extends Section<T>> SectionType getLast(SectionKey<T, Sec
       return null;
     }
 
+    public <T, SectionType extends Section<T>> Builder add(SectionBuilder<T, SectionType> builder, int firstLineIndex) {
+      return null;
```

Comment:
Is returning `null` correct here? I feel like we should at least call some other implementation of `add` and return that, because that's what all the other `add` methods do.

I also don't see this method called anywhere, so maybe we can remove it.

Reply:
Method removed.